src/azure-cli-core/setup.py

```diff
@@ -72,7 +72,9 @@
     'six',
     'tabulate>=0.7.7',
     'wheel',
-    'azure-mgmt-resource==2.1.0'
+    'azure-mgmt-resource==2.1.0',
```

Comment:
In general, our setup.py files should contain a range of acceptable versions, instead of one pinned version. Not doing this causes issues like https://github.com/Azure/azure-cli/issues/6973.

Reply:
makes sense! do you think this could be related to the alpine linux problem?